common/config/azure-pipelines/npm-publish-tarballs.yaml

```diff
@@ -0,0 +1,63 @@
+parameters:
+  - name: TestRun
+    displayName: 'Test Run (Will Not Create Tar Files for Rushstack)'
+    type: boolean
+    default: true
+
+variables:
+  - name: FORCE_COLOR
+    value: 1
+  - name: SourceBranch
+    value: $[ replace(replace(resources.repositories.self.ref, 'refs/heads/', ''), 'refs/pull/', 'refs/remotes/pull/') ]
+  - name: Publish
+    ${{ if eq(parameters.TestRun, false) }}:
+      value: '--publish'
+    ${{ else }}:
+      value: ''
+
+resources:
+  repositories:
+    - repository: 1esPipelines
+      type: git
+      name: 1ESPipelineTemplates/1ESPipelineTemplates
+      ref: refs/tags/release
+
+extends:
+  template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+  parameters:
+    pool:
+      name: Azure-Pipelines-1ESPT-ExDShared
+      os: windows
+    stages:
+      - stage:
+        jobs:
+          - job:
+            pool:
+              name: publish-rushstack-tarballs
+              os: linux
+            templateContext:
+              outputs:
+                - output: pipelineArtifact
+                  targetPath: $(Build.ArtifactStagingDirectory)/published-versions
+                  artifactName: published-versions
```

Comment:
This is how you publish a pipeline artifact. There should be another entry like this pointing to the output folder from the `rush publish --pack` command; that will produce your artifact for consumption in the separate publish pipeline.